tap_google_analytics/sync.py

```diff
@@ -88,11 +88,26 @@ def report_to_records(raw_report):
     "ga:date": '%Y%m%d',
 }
 
+def parse_datetime(field_name, value):
+    """
+    Handle the case where the datetime value is not a valid datetime format.
+
+    Google will return `(other)` as the value when the underlying database table
+    from which the report is built reaches its row limit.
+
+    See https://support.google.com/analytics/answer/9309767
+    """
+    try:
+        return datetime.strptime(value, DATETIME_FORMATS[field_name]).strftime(singer.utils.DATETIME_FMT)
+    except ValueError:
+        LOGGER.warning("Datetime value is not in expected format. It will not be transformed.")
```

Comment:
This log line might be confusing to users as written, though I agree it might be good to let folks know when this happens. Would be more helpful to log the report name and the link to the google-analytics docs here, or return 2 values (something like `parsed_datetime, row_limit_reached`?) and log it once per row instead of once per field.